layouts/shortcodes/privacy.html

```diff
@@ -0,0 +1,22 @@
+{{ $data := index site.Data site.Language.Lang }}
+
+{{ if $data.privacy.privacy.enable }}
+{{ with $data.privacy.privacy }}
+
+<section class="max-w-7xl w-full mx-auto mt-12 px-4 sm:px-6 grid lg:grid-cols-2 gap-12 lg:gap-24">
```

Comment:
The style and layout is the same as the human in the loop section. This is more of a note to remind myself to create a follow on story to move the HTML and styles to a partial so that it can be easily reused.